examples/webhooks/webhooks_test.go

```diff
@@ -9,18 +10,28 @@ import (
 )
 
 // Example handler func for processing a single event type (AccountCreated)
+//
+// The handler will only set a non-200 status if an unexpected error occurs, and
+// we want the webhook to be retried.
 func ExampleHandler_SingleEvent() {
 	var handler http.HandlerFunc = func(w http.ResponseWriter, r *http.Request) {
 		secret := "your-webhook-signing-secret" // fetched from secure storage
+
 		event, err := mhooks.ParseEvent(r, secret)
-		if err != nil {
+		if errors.Is(err, mhooks.ErrInvalidSignature) {
+			fmt.Printf("parsing event: invalid signature: %v\n", err)
+			w.WriteHeader(200)
```

Comment:
If we can't verify the signature it should return a non-200, right?

Reply:
Mmm this depends, this should virtually never error unless the user provides the wrong secret. So I guess in that case, it would be helpful for our system to retry if they correct the secret before the next retry. I can update this.